Src/Sarif.PartnerMatcher.Function/Properties/ServiceDependencies/myspamcheckertest - Zip Deploy/profile.arm.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
remove this file, since we have the second one

Reply:
deleted